source/Engine.cpp

```diff
@@ -1488,7 +1488,12 @@ void Engine::CalculateStep()
 	Point newCenterVelocity;
 	if(flagship)
 	{
-		newCenter = flagship->Position();
+		if(flagship->IsThrusting())
+			offsetMultiplier = offsetMultiplier >= 1 ? 1.0 : offsetMultiplier + 0.0025;
+		else
+			offsetMultiplier = offsetMultiplier <= 0 ? 0.0 : offsetMultiplier - 0.0025;
+		offset = flagship->Velocity() * 20 * offsetMultiplier;
+		newCenter = Preferences::Has("Camera Acceleration") ? flagship->Position() + offset : flagship->Position();
```

Comment:
Too long as well I recon (I disagree with that rule but hey I dont make the rules here)

Reply:
done